webpack.common.js

```diff
@@ -0,0 +1,26 @@
+const path = require('path');
+
+module.exports = {
+  entry: './src/index.js',
+  output: {
+    path: path.resolve(__dirname, 'dist'),
+    libraryTarget: 'umd',
+    library: 'LinxBanner',
+    globalObject: 'typeof self !== \'undefined\' ? self : this',
```

Comment:
This approach was used to keep it similar to webpack at version 3, it is really needed to keep this version working on browser and node.

This bug is better described [here](
https://github.com/webpack/webpack/issues/6522) and the reason of this change on webpack can be found [here](https://github.com/webpack/webpack/issues/6525).